setuptools_rust/utils.py

```diff
@@ -6,15 +6,29 @@
 import semantic_version
 
 
-def cpython_feature(ext=True, pyo3=False):
+class Binding:
```

Comment:
Since `setuptools-rust` supports Python 2 & 3, better use `class Binding(object):` to get ride of old style class behavior even though that don't really matter here. :-)